hawkbit-security-integration/src/main/java/org/eclipse/hawkbit/security/ControllerPreAuthenticatedSecurityHeaderFilter.java

```diff
@@ -117,12 +126,16 @@ public HeaderAuthentication getPreAuthenticatedCredentials(final TenantSecurityT
      * It's ok if we find the the hash in any the trusted CA chain to accept
      * this request for this tenant.
      */
-    private String getIssuerHashHeader(final TenantSecurityToken secruityToken, final String knownIssuerHash) {
+    private String getIssuerHashHeader(final TenantSecurityToken secruityToken, final String knownIssuerHashes) {
+        // there may be several knownIssuerHashes configured for the tenant
```

Comment:
In my opinion you can remove the comment, if you rename the `splitMultiHash` methode (see next comment)